src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -28,20 +28,7 @@
 typedef ... X509_STORE;
 typedef ... X509_VERIFY_PARAM;
 
-typedef struct x509_store_ctx_st X509_STORE_CTX;
-struct x509_store_ctx_st {
-    X509_STORE *ctx;
-    int current_method;
-    X509 *cert;
-    Cryptography_STACK_OF_X509 *untrusted;
-    Cryptography_STACK_OF_X509_CRL *crls;
-    X509_VERIFY_PARAM *param;
-    void *other_ctx;
-    int (*verify)(X509_STORE_CTX *);
-    int (*verify_cb)(int, X509_STORE_CTX *);
-    int (*get_issuer)(X509 **, X509_STORE_CTX *, X509 *);
-    ...;
-};
+typedef ... X509_STORE_CTX;
```

Comment:
It looks like 16.0 doesn't use this (The X509StoreContext object appears to treat it as opaque)